### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by vivekgudipudi**

### CODE REVIEW

**Comment by GuptaEesh** [comment]
Great

**Comment by harshitadaswani** [comment]
Looks good!

**Comment by prathmeshjagtap** [comment]
cool

**Comment by tanishk15sharma** [comment]
helloo guys

**Comment by SanjayJatti** [comment]
its awesome

File: readme.md [hunk 1]
```diff
@@ -1 +1 @@
-hey!
\ No newline at end of file
+hey! I'm vivek
```
Comment (kedark152) [inline readme.md:1]: great

**Comment by vaibhav18matere** [comment]
you can refr actor codeto ES6.

**Comment by Kavita539** [comment]
great

**Comment by ghost** [comment]
great

File: readme.md (hunk 1 above)
Comment (vivekgudipudi) [reply readme.md:1]: thanks kedark

Comment (rkapoor10) [inline readme.md:1]: Hi! I'm Ritik. Nice to meet you.

Comment (vivekgudipudi) [reply readme.md:1]: hey

**Comment by hussiantabha** [comment]
Great

**Comment by RahulSingh9131** [comment]
great!!

**Comment by Avin008** [comment]
great

**Pull request merged**